ui/appui-react/src/appui-react/accudraw/AccuDrawDialog.tsx

```diff
@@ -61,7 +61,7 @@ export function AccuDrawDialog(props: AccuDrawDialogProps) {
       dialogId={props.dialogId}
       width={dialogWidth}
       minHeight={75}
-      inset={false}
+      inset={true}
```

Comment:
Yes, no idea why inset prop is a thing even, since everything looks so much worse without inset. After my changes movable Dialogs with inset disabled just don't look right, so I had to make this change here.

Reply:
Fixed.